core-bundle/src/Resources/contao/templates/elements/ce_vimeo.html5

```diff
@@ -6,7 +6,23 @@
     <?php if ($this->aspect): ?>
       <div class="responsive ratio-<?= $this->aspect ?>">
     <?php endif; ?>
-    <iframe<?= $this->size ?> src="<?= $this->src ?>" allowfullscreen></iframe>
+      <?php if ($this->videoSplash): ?>
+        <figure id="videoSplash<?= $this->id ?>" class="image_container" data-video="<?= $this->src ?>">
```

Comment:
```suggestion
        <a id="videoSplash<?= $this->id ?>" class="image_container" href="<?= $this->src ?>">
```

Using a link would provide a perfect fallback if the JS fails IMO. Also the `click` event is more accessible on `<a>` and `<button>` elements (e.g. can be reached by keyboard navigation).

Reply:
I added some change requests. The rest of the JavaScript looks good to me.